test/fixtures/args.js

```diff
@@ -1,9 +1,8 @@
-function args(...args) {
+function args(options) {
```

Comment:
FYI: Without this change, the following test fails: `loader › should call the function with the loader options` with:
`TypeError: Converting circular structure to JSON`.

I.e. the `loaderContext` can't matched with a snapshot as it can't be serialized to JSON.